exchanges/kucoin/kucoin_wrapper.go

```diff
@@ -341,6 +341,40 @@ func (ku *Kucoin) UpdateTradablePairs(ctx context.Context, forceUpdate bool) err
 	return nil
 }
 
+// GetFuturesTickers returns the tickers for all currency pairs of the futures asset type
```

Comment:
Probably note that this will do n * REST requests based on enabled pairs.

Reply:
description amended